buildkite/scripts/rosetta-integration-tests.sh

```diff
@@ -154,6 +164,43 @@ done
 echo "==================== UNLOCKING GENESIS ACCOUNTS ======================"
 mina-dev accounts unlock --public-key $BLOCK_PRODUCER_PK
 mina-dev accounts unlock --public-key $SNARK_PRODUCER_PK
+mina-dev accounts unlock --public-key $FAUCET_PK
+
+# Start sending payments
+send_payments() {
+  mina-dev client send-payment -rest-server http://127.0.0.1:${MINA_GRAPHQL_PORT}/graphql -amount 1 -nonce 0 -receiver $BLOCK_PRODUCER_PK -sender $BLOCK_PRODUCER_PK
+  while true; do
+    sleep $TRANSACTION_FREQUENCY
+    mina-dev client send-payment -rest-server http://127.0.0.1:${MINA_GRAPHQL_PORT}/graphql -amount 1 -receiver $BLOCK_PRODUCER_PK -sender $BLOCK_PRODUCER_PK
+  done
+}
+send_payments &
+
+# Deploy zkApps
+for zkapp_path in {}; do # TODO: what's the best way to bundle the zkapps here?
```

Comment:
I have several dirs with zkApps that I want to iterate and deploy here. What should be the best way to bundle them in the CI? @kaozenn @simisimis